### ISSUE TITLE
When focusing hover, I don't hear a hint

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#188748]
Testing #188537

I cannot verify this step:

> You should hear a hint about opening the notification in the accessible view.

When focusing the hover, the screenreader announces this:

> (property) TrackingTokenizationStateStore<TState extends IState>._invalidEndStatesLineNumbers: RangePriorityQueueImpl  tool tip

**Comment by roblourens** [comment]
Sometimes I get text like that, and sometimes I _only_ get "you are currently on a tooltip" and nothing else

**Comment by meganrogge** [comment]
Do you both of screen reader mode enabled?

**Comment by hediet** [comment]
Yes

**Comment by meganrogge** [comment]
I believe you are trying this via mouse hover and click. 

I should've specified that this will only work when navigating via keyboard - as that is what most screen reader users do.

Screen reader users will access the hover via `ctrl + k + i` or `cmd +k +i`. that's when you'll hear this.

**Comment by meganrogge** [comment]
For low vision users, maybe it would be helpful to support this for tooltips as well, but it's not a big priority since most SR users navigate via the keyboard

**Comment by roblourens** [comment]
I was using the keyboard shortcut to show the hover

**Comment by meganrogge** [comment]
Maybe I'm setting the wrong aria label
